app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+class Animal(object):
+    alive = []
+
+    def __init__(self, name: str, *args: int) -> None:
+        self.name = name
+        self.hidden = False
+        if len(args) != 0:
+            self.health = args[0]
+        else:
+            self.health = 100
+        Animal.alive.append(self)
+
+    def __str__(self) -> str:
+        return f"\nName = {self.name},\n" \
+               f"Current hp = {self.health},\n" \
+               f"Hiding? = {'Yes' if self.hidden == True else 'No'}\n"
```

Comment:
Use brackets instead of \
```suggestion
        return ("\nName = {self.name},\n" 
               f"Current hp = {self.health},\n" 
               f"Hiding? = {'Yes' if self.hidden == True else 'No'}\n"
           )
```

Reply:
Fix in repr also